src/cryptography/py.typed

```diff
@@ -0,0 +1 @@
+
```

Comment:
Does the file have to empty? In the past some zip tools had issues with 0 byte files.

Reply:
```suggestion
```
I intended for this file to be 0 bytes

Reply:
Yes, let's make this a proper 0-byte file (that's what we use on pyca/bcrypt)